types/node-cron/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for node-cron 2.0
+// Type definitions for node-cron 3.0
 // Project: https://github.com/node-cron/node-cron, https://github.com/merencia/node-cron
 // Definitions by: morsic <https://github.com/maximelkin>,
 //                 burtek <https://github.com/burtek>,
 //                 Richard Honor <https://github.com/RMHonor>
 //                 Ata Berk YILMAZ <https://github.com/ataberkylmz>
+//                 Alex Seidmann <https://github.com/aseidma>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import { Timezone } from 'tz-offset';
```

Comment:
probably replace with just string, not sure if tz-offset matches with moment-timezone